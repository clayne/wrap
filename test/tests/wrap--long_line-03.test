wrap | /dev/null | | long_line-03.txt | 0
