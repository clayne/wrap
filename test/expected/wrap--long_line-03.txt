X
ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ
end
